debian/manpages/snapcraft.1

```diff
@@ -0,0 +1,87 @@
+.TH snapcraft 1 "19 July 2016" "version 2.13"
+.SH NAME
+Snapcraft - Snapcraft is a delightful packaging tool
+.SH SYNOPSIS
+snapcraft [options] <command>
+
+snapcraft (-h | --help)
+
+snapcraft --version
+.SH DESCRIPTION
+Snapcraft helps you assemble a whole project in a single tree out of many pieces. It can drive a very wide range of build and packaging systems, so that you can simply list all the upstream projects you want and have them built and installed together as a single tree.
+.SH OPTIONS
+--help
+    Prints the available Snapcraft commands.
+
+--version
+    Prints the version of Snapcraft currently installed on your system.
```

Comment:
Don't manually format definition lists with spaces this way (if you ever find yourself typing leading spaces on a line in troff, you're probably doing something wrong).  Do this instead:

    .TP
    .B \-\-help
    Prints the available Snapcraft commands.
    .TP
    .B \-\-version
    Prints the version of Snapcraft currently installed on your system.

The same goes for other such lists below.